lineapy/plugins/base_pipeline_writer.py

```diff
@@ -402,244 +395,41 @@ def write_pipeline_files(self) -> None:
         self._write_docker()
         self._create_test()
 
-
-class AirflowPipelineWriter(BasePipelineWriter):
-    """
-    Class for pipeline file writer. Corresponds to "AIRFLOW" framework.
-    """
-
-    @property
-    def docker_template_name(self) -> str:
-        return "airflow_dockerfile.jinja"
-
-    def _write_dag(self) -> None:
-        dag_flavor = self.dag_config.get(
-            "dag_flavor", "PythonOperatorPerArtifact"
-        )
-
-        # Check if the given DAG flavor is a supported/valid one
-        if dag_flavor not in AirflowDagFlavor.__members__:
-            raise ValueError(
-                f'"{dag_flavor}" is an invalid airflow dag flavor.'
-            )
-
-        # Construct DAG text for the given flavor
-        if (
-            AirflowDagFlavor[dag_flavor]
-            == AirflowDagFlavor.PythonOperatorPerSession
-        ):
-            full_code = self._write_operator_per_session()
-        elif (
-            AirflowDagFlavor[dag_flavor]
-            == AirflowDagFlavor.PythonOperatorPerArtifact
-        ):
-            full_code = self._write_operator_per_artifact()
-
-        # Write out file
-        file = self.output_dir / f"{self.pipeline_name}_dag.py"
-        file.write_text(prettify(full_code))
-        logger.info(f"Generated DAG file: {file}")
-
-    def _write_operator_per_session(self) -> str:
+    def get_pipeline_args(self) -> Dict[str, InputVariable]:
         """
-        This hidden method implements Airflow DAG code generation corresponding
-        to the `PythonOperatorPerSession` flavor, where each session gets its
-        own Python operator. For instance, if the two artifacts in our pipeline
-        (e.g., model and prediction) were created in the same session, we would
-        get an Airflow DAG file looking as the following:
-
-        .. code-block:: python
-        def dag_setup():
-            pickle_folder = pathlib.Path("/tmp").joinpath("g2_z")
-            if not pickle_folder.exists():
-                pickle_folder.mkdir()
-
-
-        def dag_teardown():
-            pickle_files = pathlib.Path("/tmp").joinpath("g2_z").glob("*.pickle")
-            for f in pickle_files:
-                f.unlink()
-
-            def task_run_session_including_g2():
-                artifacts = g2_z_module.run_session_including_g2()
-                pickle.dump(artifacts["g2"], open("/tmp/g2_z/artifact_g2.pickle", "wb"))
-                pickle.dump(artifacts["z"], open("/tmp/g2_z/artifact_z.pickle", "wb"))
-
-            with DAG(...) as dag:
```

Comment:
this piece of example might be important to guide people on what flavors mean. maybe keep a simpler version of this if its too much.

Reply:
Yes, Adding this back to _write_operators for airflow.